### ISSUE TITLE
the test model's accuracy

### ISSUE THREAD

**Issue by lassloss** [shiming-chen/MSDN#9]
I‘ve used the test models and parameters provided, but the answers' accuracy is about 10%. So I wonder the codes provided are wrong, or it's my operation fault?

**Comment by shiming-chen** [comment]
> 

Dear, @lassloss 

Thank you for interesting in our works. We can implemente the results in the paper, I guess you may incorrectly load the pre-trained models (we should note that the CZSL and GZSL models are individual for each dataset.). Please check the code carefully. 

If you have any other doubts, please don't hesistate to contact us.

Best,

Shiming
